src/Microsoft.AspNet.Diagnostics.Elm/ElmScope.cs

```diff
@@ -67,10 +68,22 @@ public static IDisposable Push([NotNull] ElmScope scope, [NotNull] ElmStore stor
             Current = scope;
             Current.Parent = temp;
 
+            // Get the scope message
+            string scopeMessage = null;
+            var scopeLogValues = Current._state as ILogValues;
+            if(scopeLogValues != null)
+            {
+                scopeMessage = scopeLogValues.Format();
```

Comment:
/cc @loudej I think the plan here to get rid of `.Format()` and just have `.ToString()` do the right thing. Should we do that change instead?